dvd/dvd_dump.ixx

```diff
@@ -715,7 +755,7 @@ export bool redumper_dump_dvd(Context &ctx, const Options &options, DumpMode dum
                 }
                 else
                 {
-                    // Make sure we don't blow past the end of the ss range when zeroing
+                    // skip at most to the end of the range
```

Comment:
Personally I think end of the range is a bit vague, I think it should probably mention `ss range` or `security sector range` or something along those lines.